### PR TITLE
Departmental Request Consoles can now make multi-line announcements too

### PR DESCRIPTION
**Pull request by ValkyrieSkies**
Departmental request consoles can now make multi-line announcements, as requested by DJ in #32435
:cl:
 * tweak: Departmental Request Consoles can now also make multi-line announcements

### CODE REVIEW

**Comment by WindowsErrors** [comment]
as a reference to future readers

you can rename the requests consoles

Screenshot (1721)

greymlins assemble

File: code/game/machinery/requests_console.dm [hunk 1]
```diff
@@ -266,15 +266,15 @@ var/list/obj/machinery/requests_console/requests_consoles = list()
...
+			to_chat(usr, "<span class='warning'>Invalid characters or no text detected.</span>")
 			dpt = "";
 			msgVerified = ""
 			msgStamped = ""
 			screen = 0
 			priority = -1
 
 	if(href_list["writeAnnouncement"])
-		var/new_message = copytext(reject_bad_text(input(usr, "Write your message:", "Awaiting Input", "")),1,MAX_MESSAGE_LEN)
+		var/new_message = copytext(stripped_message(usr, "Write your message:", "Departmental Announcement", ""),1,MAX_MESSAGE_LEN)
```
Comment (DamianX) [inline code/game/machinery/requests_console.dm]:
```suggestion
		var/new_message = stripped_message(usr, "Write your message:", "Departmental Announcement", "")
```

Comment (ValkyrieSkies) [reply code/game/machinery/requests_console.dm]: Isn't it done that way to enforce a character limit?

Comment (DamianX) [reply code/game/machinery/requests_console.dm]: `stripped_message` already does that

Comment (ValkyrieSkies) [reply code/game/machinery/requests_console.dm]: based

**Pull request merged**